webauthn-rs-proto/src/tests.rs

```diff
@@ -0,0 +1,8 @@
+use crate::RegistrationExtensionsClientOutputs;
+
+#[test]
+fn parse_cred_props() {
+    let input = r#"{"credProps":{"rk":false}}"#;
+
+    let _parsed_extn: RegistrationExtensionsClientOutputs = serde_json::from_str(input).unwrap();
```

Comment:
To test this change, shouldn't this parse a `RegisterPublicKeyCredential` or `PublicKeyCredential` JSON with the with both names for the `extensions` field?

Reply:
Yeah, the original hypothesis was something else, that's why the test was like this, I'll add the test.